pom.xml

```diff
@@ -15,6 +15,7 @@
     <module>deployment</module>
     <module>runtime</module>
     <module>docs</module>
+    <module>qute-web-markdown</module>
```

Comment:
I'd call the module just `markdown`.

Reply:
qute-markdown?

Reply:
+1 for `markdown` as the module directory name and the current `quarkus-qute-web-markdown` as the artifact id.

I will send a PR with the `core` dir where the `runtime` and `deployment` modules will be moved.

Reply:
I got it
+1